### ISSUE TITLE
map50

### ISSUE THREAD

**Issue by Ellohiye** [ultralytics/ultralytics#21632]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Is yolov8 doing experiments on voc dataset? What is the approximate detection accuracy?
How do the Yolo series divide the data sets by experimenting on the coco data set? Why is the evaluation indicator map50 val? Is it the test result obtained on the verification set?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Ellohiye, thanks for opening an issue and for your interest in Ultralytics 🚀! This is an automated response to help you get started. An Ultralytics engineer will follow up with additional assistance soon.

We recommend a visit to the Docs for background and examples, including many Python and CLI usage patterns that may address your questions:
- Docs: https://docs.ultralytics.com/
- Python examples: https://docs.ultralytics.com/usage/python/
- CLI examples: https://docs.ultralytics.com/usage/cli/

If this is a 🐛 Bug Report, please provide a minimum reproducible example (MRE) to help us debug it: https://docs.ultralytics.com/help/minimum-reproducible-example/

If this is a custom training ❓ Question, please include as much detail as possible:
- Exact command(s) or code used
- Model and dataset details (e.g., dataset split, annotations format)
- Relevant logs and screenshots
- Confirm you’re following our Tips for Best Training Results: https://docs.ultralytics.com/guides/model-training-tips/

Join the Ultralytics community where it suits you best:
- Real-time chat on Discord 🎧: https://discord.com/invite/ultralytics
- In-depth discussions on Discourse: https://community.ultralytics.com/
- Share insights on our Subreddit: https://www.reddit.com/r/Ultralytics/

## Upgrade

Upgrade to the latest ultralytics package including all requirements in a Python>=3.8 environment with PyTorch>=1.8 to verify your issue isn’t already resolved in the latest version:

```bash
pip install -U ultralytics
```

Requirements: https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including CUDA/CUDNN, Python and PyTorch preinstalled):

- Notebooks with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- Google Cloud Deep Learning VM. See GCP Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/
- Amazon Deep Learning AMI. See AWS Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/
- Docker Image. See Docker Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/ <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all Ultralytics CI tests are currently passing. CI tests verify correct operation of all YOLO Modes and Tasks on macOS, Windows, and Ubuntu every 24 hours and on every commit.

To help us address your specific questions about datasets, metrics (e.g., mAP50), and splits, please share:
- Which model(s) you’re referring to (e.g., YOLO11, YOLOv10, etc.)
- The exact dataset and split protocol you’re using (e.g., COCO train/val/test, VOC year/version)
- The commands or config used for training/validation and any resulting logs or metrics 📊

Thanks again!

**Comment by Y-T-G** [comment]
We don't have evaluation metrics in VOC dataset.

Ultralytics reports the standard COCO evaluation metrics for YOLO models using COCOEval which provides mAP50-95 and mAP50.

For MS-COCO evaluation, the val2017 split is used for evaluation.

**Comment by Ellohiye** [comment]
Shouldn't COCO dataset compare detection accuracy in the test set?

**Comment by Y-T-G** [comment]
The COCO val2017 is used by most papers to report results

**Comment by Ellohiye** [comment]
Are you using best.pt or last.pt in your experiment to get the detection results on the coco verification set?

**Comment by Y-T-G** [comment]
`best.pt`

**Comment by Ellohiye** [comment]
How to tune hyperparameters

**Comment by glenn-jocher** [comment]
Tune on your train/val split by iterating LR schedule, image size, batch, and augmentations, and pick settings by highest mAP50-95 on val; see Model Validation with Ultralytics YOLO for measuring changes and Insights on Model Evaluation and Fine-Tuning for practical tuning tips.
